packages/smart-contracts/src/contracts/ERC20TransferableReceivable.sol

```diff
@@ -158,15 +158,16 @@ contract ERC20TransferableReceivable is ERC721, ERC721Enumerable, ERC721URIStora
    * @param paymentReference A reference for the payment.
    * @param amount The amount of ERC20 tokens to be paid.
    * @param erc20Addr The address of the ERC20 token to be used as payment.
-   * @param newTokenURI The URI to be set on the minted receivable token.
+   * @param requestID The ID of the request associated with the receivable.
+   *                  Useful for retrieving details of the request from the Request Network.
    * @dev Anyone can pay for the mint of a receivable on behalf of a user
    */
   function mint(
     address owner,
```

Comment:
Change Request: 

I find the `owner` argument name confusing. `initialOwner` would be more clear.

Reply:
I'm torn about it, but I guess I'm fine with not making this change given that it would have ripple effects through the payment-processing package.